cmd/controller-manager/app/controllers.go

```diff
@@ -348,6 +352,36 @@ func addAllControllers(mgr manager.Manager, client k8s.Client, informerFactory i
 		addController(mgr, "volumesnapshot", volumeSnapshotController)
 	}
 
+	// "pvc-autoresizer"
+	if cmOptions.IsControllerEnabled("pvc-autoresizer") {
+		if err := runners.SetupIndexer(mgr, false); err != nil {
+			return err
+		}
+
+		promClient, err := runners.NewPrometheusClient("http://prometheus-k8s.kubesphere-monitoring-system:9090")
```

Comment:
is the url `http://prometheus-k8s.kubesphere-monitoring-system:9090` fixed? hard code is not a good idea. at least it should be constructed with some constants.

Reply:
Okay，I‘ll fix it.